packages/scripts/config/webpack.config.js

```diff
@@ -232,7 +232,7 @@ const config = {
 		new CopyWebpackPlugin( {
 			patterns: [
 				{
-					from: copyWebPackPattens,
+					from: copyWebPackPatterns,
```

Comment:
NIt: it's webpack not WebPack 😄

Reply:
🤦 Nice catch - I've updated the reference.